vendor/bat-native-ledger/src/bat/ledger/internal/contribution/phase_one.cc

```diff
@@ -9,6 +9,7 @@
 #include "bat/ledger/internal/contribution/phase_one.h"
 #include "bat/ledger/internal/contribution/phase_two.h"
 #include "bat/ledger/internal/ledger_impl.h"
+#include "bat/ledger/internal/uphold/uphold_util.h"
```

Comment:
could we move `ConvertToProbi` to more general util (new util file) if it's used outside `uphold` folder. So that we don't include `uphold` folder into `contribution` folder.

Reply:
Fixed!